### PR TITLE
Update smoke tests plugin versions

### PR DESCRIPTION
**Pull request by ov7a**
### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by ov7a** [comment]
@bot-gradle help

**Comment by ov7a** [comment]
@bot-gradle test AST

File: testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy [hunk 1]
```diff
@@ -185,16 +185,16 @@ abstract class AbstractSmokeTest extends Specification {
...
 
         // https://github.com/etiennestuder/teamcity-build-scan-plugin/releases
         static teamCityGradlePluginRef = "v0.35"
 
         // https://github.com/jenkinsci/gradle-plugin/releases
-        static jenkinsGradlePluginRef = "gradle-2.9"
+        static jenkinsGradlePluginRef = "gradle-2.10"
 
         // https://github.com/gradle/gradle-enterprise-bamboo-plugin/releases
         static bambooGradlePluginRef = "gradle-enterprise-bamboo-plugin-1.2.0"
```
Comment (ov7a) [inline testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy:200]: See https://github.com/gradle/gradle/pull/28673/files#r1557300152

**Comment by ov7a** [comment]
@bot-gradle test AST

**Review by ljacomet — APPROVED** [review]

**Comment by ov7a** [comment]
@bot-gradle test RfN

**Pull request merged**